skimage/metrics/simple_metrics.py

```diff
@@ -259,3 +263,51 @@ def normalized_mutual_information(image0, image1, *, bins=100):
     H01 = entropy(np.reshape(hist, -1))
 
     return (H0 + H1) / H01
+
+
+def enhancement_measure(image, size=3, eps=1e-6):
+    """Compute the enhancement measure of an image.
+
+    The enhancement measure (EME) of an image, aims to provide a perceptual
+    estimate of image quality, based on the luminance ratio within a sliding
+    window [1]_. The resulting measure is useful when comparing image
+    enhancement algorithms.
+
+    Parameters
+    ----------
+    image : array
+        Input image of which the quality should be assessed.
+        Can be either 3-channel RGB or 1-channel grayscale.
```

Comment:
We should probably support only grayscale images as input to make sure that the user determines [how the total luminance of a colored image is calculated](https://scikit-image.org/docs/stable/api/skimage.color.html#skimage.color.rgb2gray). Currently, the way that `maximum_filter` and `minimum_filter` is combining color channels is not something that is covered by the paper.